ee_tests/Dockerfile.builder

```diff
@@ -34,6 +36,7 @@ RUN yum  --setopt tsflags='nodocs' -y update && \
       xorg-x11-server-Xvfb google-chrome-stable && \
       yum -y clean all
```

Comment:
Does it make sense to clean yum data? It will make a resulting image a little bit smaller but IMHO it's not worth it.